src/lib.rs

```diff
@@ -78,7 +78,7 @@ impl Key for UrlEncodedQuery {
     type Value = QueryMap;
 }
 
-impl<'a, 'b> plugin::Plugin<Request<'a, 'b>> for UrlEncodedQuery {
+impl plugin::Plugin<Request> for UrlEncodedQuery {
     type Error = UrlDecodingError;
 
     fn eval(req: &mut Request) -> QueryResult {
```

Comment:
`req.url.query()` seems to fail or something, at least in my testrun it did not give me any data …